### PR TITLE
Update functional tests for /firefox/new/ page

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
Updates functional tests for `/firefox/new` since, we refactored the page into two separate templates (see [bug 1251304](https://bugzilla.mozilla.org/show_bug.cgi?id=1251304) 

## Checklist
- [ ] Requires l10n changes.
- [x] Related functional & integration tests passing.

@davehunt r?

### CODE REVIEW

**Comment by alexgibson** [comment]
Note: you can ignore the circleci build failure below for now (see #3926)

File: tests/functional/firefox/new/test_download.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+@pytest.mark.smoke
+@pytest.mark.sanity
+@pytest.mark.nondestructive
+def test_download_button_displayed(base_url, selenium):
+    page = DownloadPage(base_url, selenium).open()
+    assert page.download_button.is_displayed
+
+
+@pytest.mark.skip_if_firefox
+@pytest.mark.skip_if_internet_explorer(reason='https://github.com/SeleniumHQ/selenium/issues/448')
```
Comment (davehunt) [inline tests/functional/firefox/new/test_download.py]: reason is a nice touch, we should check for it and include it in the skip message in https://github.com/mozilla/bedrock/blob/master/tests/functional/conftest.py#L28-L35

**Comment by davehunt** [comment]
r+

**Comment by alexgibson** [comment]
@davehunt - added `reason` kwarg to the skip message :smile:

**Comment by davehunt** [comment]
Nice! r+

**Pull request merged**